### ISSUE TITLE
PRD: Make font-size adjustable 

### ISSUE THREAD

**Issue by saleweaver** [sourcegraph/jetbrains#2461]
### Description:

As a user, I want to be able to read the chat output, even if I'm using a large monitor/5k resolution. Currently the font size is fixed to 13px from the cody web view, and there's no configuration option. My editor font is set to 17px, but the plugin's font size is not adapting to the editor font size changes. The ideal way IMHO would be to either add a configuration option for font size, or to make the browser's behaviour of cmd + '+' working. 

![Screenshot 2024-10-16 at 10 25 59](https://github.com/user-attachments/assets/d50377c1-6ae8-4bed-858c-04d78fd2ecb1)

### Objectives:

The goal is to be able to use the plugin and to be able to read the chat output clearly, for different screen sizes and resolutions

### Acceptance Criteria:

- The user should be able to adapt font size to their needs
- The user should be able to read the chat content

### Design Tasks:

```[tasklist]
### Design Tasks
- [ ] Add a draft title or issue reference here
```

### Engineering Tasks:

```[tasklist]
### Engineering Tasks
- [ ] Add a draft title or issue reference here
```

### Related Docs:

- [PRD: Your-Title]()
